examples/text-generation/quantization_tools/postprocess_measure.py

```diff
@@ -0,0 +1,145 @@
+        fsdpa_v_input = None
+        # OH
+        oh_k_cache_input = None
+        oh_v_cache_input = None
+        
+        attn_name = "attn"
+        k_cache_name = "k_cache"
+        v_cache_name = "v_cache"
+        if args.deepseek:
+            print(f"Handling deepseek model")
+            attn_name = "mla_attn"
+            k_cache_name = "latent_cache_k"
+
+        for node_name, node_info in json_data['Nodes'].items():
+            if f'model.layers.{layer_index}.self_attn.{attn_name}.impl.matmul_av' in node_name:
+                matmul_av_input = node_info['inputs'][1]
+            if f'model.layers.{layer_index}.self_attn.{attn_name}.impl.{v_cache_name}' in node_name:
+                v_cache_input = node_info['inputs'][0]
+            if f'model.layers.{layer_index}.self_attn.{attn_name}.impl.matmul_qk' in node_name:
+                matmul_qk_input = node_info['inputs'][1]
+            if f'model.layers.{layer_index}.self_attn.{attn_name}.impl.{k_cache_name}' in node_name:
+                k_cache_input = node_info['inputs'][0]
+            # Flash attention case
+            if f'model.layers.{layer_index}.self_attn.fused_scaled_dot_product_attention' in node_name:
+                fsdpa_k_input = node_info['inputs'][1]
+                fsdpa_v_input = node_info['inputs'][2]
+            # Optimum-habana case
+            if f'model.layers.{layer_index}.self_attn.{k_cache_name}' in node_name:
+                oh_k_cache_input = node_info['inputs'][0]
+            if f'model.layers.{layer_index}.self_attn.{v_cache_name}' in node_name:
+                oh_v_cache_input = node_info['inputs'][0]
+
+        if matmul_av_input != v_cache_input:
+            if args.deepseek:
+                # For deepseek, there is one tensor for k_cache and v_cache
+                json_data['Nodes'][f'model.layers.{layer_index}.self_attn.{attn_name}.impl.matmul_av']['inputs'][1] = k_cache_input
+            else:
+                json_data['Nodes'][f'model.layers.{layer_index}.self_attn.attn.impl.matmul_av']['inputs'][1] = v_cache_input
+        if matmul_qk_input != k_cache_input:
+            json_data['Nodes'][f'model.layers.{layer_index}.self_attn.attn.impl.matmul_qk']['inputs'][1] = k_cache_input
+        
+        # Flash attention
+        if fsdpa_k_input != oh_k_cache_input:
+            json_data['Nodes'][f'model.layers.{layer_index}.self_attn.fused_scaled_dot_product_attention']['inputs'][1] = oh_k_cache_input
+        if fsdpa_v_input != oh_v_cache_input:
+            json_data['Nodes'][f'model.layers.{layer_index}.self_attn.fused_scaled_dot_product_attention']['inputs'][2] = oh_v_cache_input
+
+    return json_data
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser(
+        description="Run the measurements parser", formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument(
+        "-m", "--measurements", type=str, help="full path to the directory of the measurements that should be fixed"
```

Comment:
```suggestion
        "-m", "--measurements", type=str, required=True, help="full path to the directory of the measurements that should be fixed"
```

It sounds to me this arg is required otherwise it will fail